R/retrieveData.R

```diff
@@ -73,7 +73,7 @@ retrieveData <- function(model, rev = 0, dev = "", cachetype = "rev", puc = iden
 
     .match <- function(folder, fileType, pattern) {
       match <- dir(path = folder, pattern = paste0(".*\\.", fileType))
-      match <- match[startsWith(match, pattern) & !startsWith(match, paste0(pattern, "_debug"))]
+      match <- match[(startsWith(match, paste0(pattern, "_")) | startsWith(match, paste0(pattern, "."))) & !startsWith(match, paste0(pattern, "_debug"))] # nolint
```

Comment:
why is the `nolint` required here?

Reply:
Lintr complained about the line lenght. I inserted a line break.